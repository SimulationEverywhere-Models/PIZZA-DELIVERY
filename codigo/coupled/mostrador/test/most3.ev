00:00:05:000 Abrir_m 1
00:00:12:000 Recibe_ent_m 1
00:00:18:000 Lleno_ent 1
00:00:22:000 Recibe_ent_m 2
00:00:30:000 Lista_ent_m 1
00:00:40:000 Atiende_m 5
00:00:53:000 Cerrar_m 1
00:00:55:000 Recibe_ent_m 5
