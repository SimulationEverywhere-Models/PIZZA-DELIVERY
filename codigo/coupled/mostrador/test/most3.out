00:00:05:000 lista_ll_m 1
00:00:12:000 ocupada_ent_m 1
00:00:12:030 dist_out_m 1
00:00:12:030 libre_m_ent 1
00:00:18:000 cerrado_m 1
00:00:22:000 ocupada_ent_m 1
00:00:22:030 dist_out_m 2
00:00:22:030 libre_m_ent 1
00:00:30:000 lista_ll_m 1
00:00:40:000 ocupado_m 1
00:00:42:000 libre_m 1
00:00:42:030 manda_pedido 5
00:00:53:000 cerrado_m 1
00:00:55:000 ocupada_ent_m 1
00:00:55:030 dist_out_m 5
00:00:55:030 libre_m_ent 1
